custom/icds_reports/utils/data_pull.sql

```diff
@@ -0,0 +1,70 @@
+CREATE TABLE tmp_daily_attendance AS
+    SELECT
+        awc_id,
+        pse_date,
+        form_location_lat,
+        form_location_long
+        FROM daily_attendance_view WHERE state_id='7fb6f3fe5e7540e7be63c848c28c97ed' AND image_name IS NOT NULL ORDER BY pse_date DESC
```

Comment:
you can put the where clause in the below query only and no need to create this temp table

Reply:
rank function won't work on a view.